atd-vzd/README.md

```diff
@@ -29,16 +28,9 @@ hasura metadata apply --envfile .env.local
 - Start the local Hasura console and make any changes needed which will then reflect in your project folder
 ### Merging an approved feature branch
 
-We need to check the status of new migrations against the staging Hasura engine **before merging a feature branch** so that we can make updates to the migration version order if needed. The version refers to the timestamp in migration folder name.
-
-To check the status from your local project folder, run:
-```bash
-hasura migrate status --envfile .env.staging
-```
-
-Check that your new migration shows `NOT PRESENT` in the `DATABASE` column and that it is also the last row in the table shown. We need to make sure that migrations are applied in the same order.
+We need to check the order of migrations against those in the `master` **before merging a feature branch** so that we can make updates to the migration version order if needed. The version refers to the timestamp in migration folder name.
```

Comment:
typo here :see_no_evil: , update to "those in master" or "those in the master branch" ?

Reply:
🙏 fixing this!